### DISCUSSION TITLE
Doris Ranger plug-in shows only 'Select' permission when creating a new policy

### DISCUSSION THREAD

**Discussion by cutebear1228** [apache/doris#42890] [A - General / Q&A]
Hi there , 
I installed the Doris Ranger plug-in for data account permission control . 
However when I create a new policy for the new account , the Permission list only shows "Select". Do I need to adjust some settings to display the complete permission list ?
![image](https://github.com/user-attachments/assets/ce61ed6e-ea09-4ca6-81f6-4e3774b2140b)

Some default policies have the complete permission list. Please see the following screenshot. I would like to understand the reason for the difference. Thanks.

![image](https://github.com/user-attachments/assets/cec2ef10-3200-409b-af8d-e7a1c157f23f)

**Comment by ixzc** [comment]
If you do not select a column, you will have other permissions, but only SELECT permissions for columns

**Comment by cutebear1228** [comment]
It works.  Thank you .  

![image](https://github.com/user-attachments/assets/19de5a75-7259-4221-abad-b6ab0d196284)

**Comment by gagan211** [comment]
hey @cutebear1228 can you explain me how you installed the ranger plugin ? i tried the same using the docs but it is not working for me 

1. i installed apache ranger from the git repo [started the admin].
2. created the folder namde doris in the plugin dir and then placed the files that were present in the docs but my ranger instance is not showing any databases or tables from doris.

**Reply by cutebear1228** [reply]
Hi @gagan211 , 
I installed the ranger plugin by following the steps in the instruction on the [website](https://doris.apache.org/docs/admin-manual/auth/ranger#configure-doris-cluster) . 
This is the folder I created for ranger and the related path for your reference.  
If it works , you can type in "*" in the ranger admin instance then it will display the content in doris database . 

![image](https://github.com/user-attachments/assets/ba5d4605-df51-4186-9ff1-3b53371b7601)

Add following files in doris fe folder , and add following configuration in the file
**access_controller_type=ranger-doris**

![image](https://github.com/user-attachments/assets/594ce9b1-d142-4787-b4a0-31e480da668d)

**Comment by gagan211** [comment]
thanks for the reply Sharon
but there is also one more thing that i would like to ask you about apache
ranger :
1. how did you install it? As i think i am doing something wrong
2. the issue that i am talking about is :- after installing and executing
the curl command , when i try to log back into the ranger it gives a 404
error on the login page .
if you can describe the procedure of ranger downloading too  ?
3. also on what basis did you mention the ranger.admin.chache.dir ? is it
something mentioned in the ranger-admin config if yes then where ? if not
then is it something that you have just created and used ?
